test/Makefile.mk

```diff
@@ -212,13 +212,17 @@ GTEST_TESTS = test/gtest_links/AccumulatorTest.empty \
               test/gtest_links/FrequencyGovernorTest.frequency_bounds_in_range \
               test/gtest_links/FrequencyGovernorTest.frequency_bounds_invalid \
               test/gtest_links/FrequencyGovernorTest.validate_policy \
+              test/gtest_links/FrequencyMapAgentTest.name \
               test/gtest_links/FrequencyMapAgentTest.adjust_platform_map \
               test/gtest_links/FrequencyMapAgentTest.adjust_platform_uncore \
-              test/gtest_links/FrequencyMapAgentTest.name \
+              test/gtest_links/FrequencyMapAgentTest.split_policy \
               test/gtest_links/FrequencyMapAgentTest.enforce_policy \
               test/gtest_links/FrequencyMapAgentTest.policy_to_json \
-              test/gtest_links/FrequencyMapAgentTest.split_policy \
```

Comment:
Please put these (the `name` and `split_policy` tests) back in the order they were previously in.  We try to keep things arranged alphabetically by class.